src/JoomlaBrowser.php

```diff
@@ -897,4 +897,32 @@ public function disableStatistics()
         $I->waitForElement(['link' => 'Never'], 60);
         $I->click(['link' => 'Never']);
     }
+    
+        /**
+     * Function to delete the menu item
+     *
+     * @param  string  $menuItem   Menu Item to be deleted
+     * @param  string  $menu    Menu type of the menu item
+     *
+     * @return void
+     */
+    public function deleteMenuItem($menuItem)
+    {
+        $I->searchForItem($title);
+        $I->click(['xpath' => "//input[@id='cb0']"]);
+        $I->click(['xpath'=> "//button[@onclick=\"if (document.adminForm.boxchecked.value==0){alert('Please first make a selection from the list.');}else{ Joomla.submitbutton('items.trash')}\"]"]);
+        $I->waitForText('Menus: Items (Main Menu)','30',['css' => 'h1']);
+        $I->expectTo('see a success message and the menu removed from the list');
+        $I->see('1 menu item successfully trashed.',['id' => 'system-message-container']);
```

Comment:
missing a waitForElement ['id' => 'system-message-container'] before this line. Otherwise you could get an error if selenium is faster than Joomla system message.